### ISSUE TITLE
Any plans to add linux support?

### ISSUE THREAD

**Issue by BRENTGAMEZ** [montoyo/webdisplays#75]
Hi I saw [here](https://montoyo.net/wdwiki/index.php/Main_Page) that linux support may be added or at least try to be ported to work and was wondering if it was still planed or any updates about it?

**Comment by JakubPlatypus** [comment]
Yes, I too would be interested in this. Especially considering that mcef supports linux in newer versions.
